pegasus/sites.v3/csedweek.org/views/map.haml

```diff
@@ -22,14 +22,18 @@
   var resizeZoom;
   var scrollwheelOption = false;
   var draggableOption = false;
+  var resizeThrottleTimeout;
 
   $(document).ready(function() {
     initializeMap();
     addGeocomplete();
+    resizeMap();
 
     $(window).resize(function() {
-      resizeMap();
-    }).trigger('resize');
+      // Throttle calling of resizeMap
+      clearTimeout(resizeThrottleTimeout);
+      resizeThrottleTimeout = setTimeout(resizeMap, 500);
```

Comment:
put 500 in a separate variable with a comment so we understand what we're doing here.